tests/sapt-dft2/input.dat

```diff
@@ -29,6 +29,6 @@ set {
 
 energy('sapt(dft)', molecule=dimer)
 
-for k, v in Eref.items():                                  # TEST
-    compare_values(v / 1000.0, psi4.get_variable(k), 6, k) # TEST
+for k, v in Eref.items():                                  #TEST
```

Comment:
A few people have made the `# TEST` mistake.  We may want to modify the regex in future to allow it.  Not related to this PR in any way, of course.